src/winter_onboarding_2021/fleet_management_service/views/cab.clj

```diff
@@ -52,14 +78,23 @@
    {:label "Created at" :value :cabs/created-at}
    {:label "Updated at" :value :cabs/updated-at}])
 
+(defn format-date [date]
+  (.format (java.text.SimpleDateFormat. "MM/dd/yyyy HH:mm") date))
+
 (defn gen-cab-row [row]
```

Comment:
What are you generating here?

Reply:
Might help to read the first `Pure functions` section here about nouns.

https://stuartsierra.com/2016/01/09/how-to-name-clojure-functions